.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
-- [ ] Add a test for the CMORized data to recipes/example/recipe_check_obs.yml and run the recipe, to make sure the CMOR checks pass without errors
-- [ ] There are clear instructions on how to obtain the data
-- [ ] Tag @remi-kazeroni in this pull request, so that the new dataset can be added to the OBS data pool at DKRZ and synchronized with CEDA-Jasmin
+## Checklist
 
-Automated checks pass, status can be seen below the pull request:
+It is the responsibility of the author to make sure the PR is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.
 
-- [ ] Circle/CI tests pass. If the tests are failing, click the `Details` link to find out why.
-- [ ] Preferably Codacy code quality checks pass, however a few remaining hard to solve Codacy issues are still acceptable. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-- [ ] The documentation is building successfully on readthedocs and looks well formatted, click the `Details` link to see it.
+- [ ] [🛠][1] PR has a descriptive title for the [changelog](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review)
+- [ ] [🛠][1] Code follows the [style guide](https://docs.esmvaltool.org/en/latest/community/introduction.html#code-style)
+- [ ] [🛠][1] [Documentation](https://docs.esmvaltool.org/en/latest/community/introduction.html#documentation) is available
+- [ ] [🛠][1] YAML files pass [`pre-commit`](https://docs.esmvaltool.org/en/latest/community/introduction.html#pre-commit) or [`yamllint`](https://docs.esmvaltool.org/en/latest/community/introduction.html#yaml) checks
+- [ ] [🛠][1] [Circle/CI tests pass](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review)
+- [ ] [🛠][1] [Codacy code quality checks pass]((https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review))
+- [ ] [🛠][1] [Documentation builds successfully](https://docs.esmvaltool.org/en/latest/community/introduction.html#Branches-pull-requests-and-code-review) on readthedocs
 
+### New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
 
-**Checklist for scientific review**
+- [ ] [🛠][1] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added
+- [ ] [🛠][1] New dependencies are added to the [project requirements](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#additional-dependencies)
+- [ ] [🧪][2] [Documentation](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#documentation) for the recipe/diagnostic clearly describes how the recipe works
+- [ ] [🧪][2] Recipe runs successfully on your own machine or [`@esmvalbot`](https://github.com/apps/esmvalbot)
```

Comment:
```suggestion
- [ ] [🧪][2] Recipe runs successfully on [`@esmvalbot`](https://docs.esmvaltool.org/en/latest/community/introduction.html#running-tests) or some other machine
```

Reply:
I think we should not use the term "own machine" as getting it to run on only your laptop is not good enough. Made it a bit more neutral. Also changed the link to the documentation section that will contain info on the bot once the Bot PR is merged.